ergo-wallet/src/main/scala/org/ergoplatform/wallet/Utils.scala

```diff
@@ -64,4 +64,16 @@ object Utils {
     )
   }
 
+  implicit class EitherOpsFor211[+A, +B](val source: Either[A, B]) extends AnyVal {
```

Comment:
@greenhat Please add ScalaDoc with more create description of this.

Reply:
Done.